spec/api-menu-spec.ts

```diff
@@ -927,18 +928,20 @@ describe('Menu module', function () {
       w.show();
     });
 
-    it('does not crash when rendering menu item with Super or meta accelerator', async () => {
-      const menu = Menu.buildFromTemplate([{
-        label: 'Test Super',
-        accelerator: 'Super+Ctrl+T'
-      }, {
-        label: 'Test Meta',
-        accelerator: 'Meta+Ctrl+T'
-      }]);
-      const menuWillClose = once(menu, 'menu-will-close');
-      menu.popup({ window: w });
-      menu.closePopup();
-      await menuWillClose;
+    it('does not crash when rendering menu item with single accelerator combinations', async () => {
+      const chunkSize = 10;
+      for (let i = 0; i < singleModifierCombinations.length; i += chunkSize) {
+        const chunk = singleModifierCombinations.slice(i, i + chunkSize);
+        const menu = Menu.buildFromTemplate([
+          ...chunk.map(combination => ({
+            label: `Test ${combination}`,
+            accelerator: combination
+          }))
+        ]);
+        menu.popup({ window: w });
+        menu.closePopup();
+        await setTimeout(100);
+      }
     });
```

Comment:
I took a look at this to see if there is a better way to handle this instead of using a setTimeout.  If we separate each chunk out into its own test, each test gets its own window (which gets cleaned up at the end of the test) and therefore removes the need to wait for the popup to close - particularly because these tests are just looking for crashes, not testing to see if the popup closes.
```suggestion
    const chunkSize = 10;
    let chunkCount = 0;
    const totalChunks = Math.ceil(singleModifierCombinations.length / chunkSize);
    for (let i = 0; i < singleModifierCombinations.length; i += chunkSize) {
      const chunk = singleModifierCombinations.slice(i, i + chunkSize);
      it(`does not crash when rendering menu item with single accelerator combinations ${++chunkCount}/${totalChunks}`, async () => {
        const menu = Menu.buildFromTemplate([
          ...chunk.map(combination => ({
            label: `Test ${combination}`,
            accelerator: combination
          }))
        ]);
        menu.popup({ window: w });
        menu.closePopup();
      });
    }
```